Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -26,7 +26,7 @@ function getCommandOutputFormat(kubecommand: string): string[] {
     var args: string[] = [];
     var outputFormat = tl.getInput("outputFormat", false);
     switch (outputFormat) {
-        case '':
+        case null:
```

Comment:
In case the user has given empty string for `outputFormat`, it would change the command to `kubectl something -o ` which is not correct. You should keep the handling for empty strings.

Reply:
Let's keep both, just to be safe.

Reply:
Done